face_track/main.py

```diff
@@ -27,9 +27,9 @@
 
 print "Start face-tracking node"
 ft = FaceTrack()
-
+ft.do_look_at_actions()
 while not rospy.is_shutdown():
-	ft.do_look_at_actions()
+	#ft.do_look_at_actions()
```

Comment:
Hi Linas,
 I will change it to rospy.spin, although python is a strange thing, ros code kind of runs in parallel even with a tight loop in python.